sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -55,8 +55,36 @@ typedef struct
    * The model ID used to identify the capabilities of a device based on the Digital Twin document.
    */
   az_span model_id;
+
+  /**
+   * The data_format is sent in the username to inform IoT Hub what format the device expects Twin
+   * Document and Direct Method payloads to use. If this option is not set, the default is JSON. If
+   * data format provided is not supported by IoT Hub, service will close the connection with error
+   * 4 code indicating Bad Username/Password.
+   */
+  az_span method_twin_content_type;
+
 } az_iot_hub_client_options;
 
+/**
+ * @brief Azure IoT Hub Client option values. Not all options will have pre-defined values.
```

Comment:
> Not all options will have pre-defined values.

What do you mean by this?

Reply:
This is the first time one of the options has macro defined options offered in the header file.  The other options don't have them.

Reply:
comment removed in doxygen fix commit.